Python/bytecodes.c

```diff
@@ -0,0 +1,3983 @@
+#include "Python.h"
+
+#include "opcode.h"
+#include "pycore_atomic.h"
+#include "pycore_frame.h"
+
+void _PyFloat_ExactDealloc(PyObject *);
+void _PyUnicode_ExactDealloc(PyObject *);
+
+#define SET_TOP(v)        (stack_pointer[-1] = (v))
+#define GETLOCAL(i)     (frame->localsplus[i])
+
+#define inst(name, stack_effect) case name:
```

Comment:
I got a nice big pile of red squiggles when I opened `bytecodes.c`. Here are all of the issues:

- The `inst` macro definition has a `stack effect` parameter that isn't used by any of the actual instructions. 
- Some labels are missing:
  - `exception_unwind`
  - `resume_frame`
  - `resume_with_error`
  - `start_frame`
- Some locals are missing:
  - `call_shape`
  - `first_instr`
  - `throwflag`
- A bunch of other stuff is missing:
  - `NAME_ERROR_MSG`
  - `PEEK`
  - `PyDictKeyEntry`
  - `PyDictOrValues`
  - `PyDictUnicodeEntry`
  - `PyModuleObject`
  - `_PyDictOrValues_GetValues`
  - `_PyListIterObject`
  - `_PyObject_DictOrValuesPointer`
  - `_PyOpcode_Deopt`
  - `_PyRangeIterObject`
  - `_Py_ID`
  - `_Py_STR`
  - `binary_ops`
  - `struct _dictkeysobject`
  - `struct _dictvalues`
  - `struct _is`

Reply:
Good point. I added some more dummy arguments and labels, fixed the `inst()` macro, and I'm just copying all `#include`s from ceval.c...